mlrun/datastore/targets.py

```diff
@@ -1189,7 +1189,7 @@ def prepare_spark_df(self, df, key_columns):
         from pyspark.sql.functions import udf
         from pyspark.sql.types import StringType
 
-        udf1 = udf(lambda x: x + "}:static", StringType())
+        udf1 = udf(lambda x: str(x) + "}:static", StringType())
```

Comment:
what about the corresponding case of storey over redis, does it need handling as well?

Reply:
No, only spark